plugins/modules/ec2_key.py

```diff
@@ -118,9 +118,9 @@
       description: private key of a newly created keypair
       returned: when a new keypair is created by AWS (key_material is not provided)
       type: str
-      sample: '-----BEGIN RSA PRIVATE KEY-----
+      sample: '-----BEGIN RSA PRIVATE KEY EXAMPLE-----
         MIIEowIBAAKC...
-        -----END RSA PRIVATE KEY-----'
+        -----END RSA PRIVATE KEY EXAMPLE-----'
```

Comment:
The wrapper here brings value (more so if it's in the other direction, but still).  While adding 'EXAMPLE' to the header deliberately breaks the match, it's kind of a half-job.  Those headers are pre-known (in the same way the `AIDA` etc prefixes are) and aren't themselves 'secret', they just tell you what kind of data to expect.

```
      sample: '-----BEGIN RSA PRIVATE KEY -----
        EXAMPLE
        -----END RSA PRIVATE KEY EXAMPLE-----'
```

Hopefully we could convince the team to explicitly whitelist `EXAMPLE` **inside** the headers as being 'ok'